tests/integration/components/system-status-test.js

```diff
@@ -0,0 +1,64 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupIntl } from 'ember-intl/test-support';
+
+module('Integration | Component | system-status', function (hooks) {
+  setupRenderingTest(hooks);
+  setupIntl(hooks);
+
+  test('it renders system status details', async function (assert) {
+    await render(hbs`<SystemStatus />`);
+
+    assert.dom('[data-test-system-status]').exists();
+
+    assert
+      .dom('[data-test-system-status-title]')
+      .exists()
+      .containsText('t:systemStatus:()');
+
+    assert.dom('[data-test-system-status-systems]').exists({ count: 3 });
+
+    const rows = this.element.querySelectorAll(
+      '[data-test-system-status-systems]'
+    );
+
+    assert.dom(rows[0]).containsText('t:storage:()');
+    assert.dom(rows[1]).containsText('t:devicefarm:()');
+    assert.dom(rows[2]).containsText('t:api:() t:server:()');
+  });
+
+  test('it renders system status operational when isWorking true', async function (assert) {
+    await render(
+      hbs`<SystemStatus::Status @isRunning={{false}} @isWorking={{true}} />`
+    );
+
+    assert
+      .dom('[system-status-status-operational]')
+      .exists()
+      .containsText('t:operational:()');
+  });
+
+  test('it renders system status unreachable when isWorking false', async function (assert) {
+    await render(
+      hbs`<SystemStatus::Status @isRunning={{false}} @isWorking={{false}} />`
+    );
+
+    assert
+      .dom('[system-status-status-unreachable]')
+      .exists()
+      .containsText('t:unreachable:()');
+  });
+
+  test('it renders system status loading when isRunning true', async function (assert) {
+    await render(
+      hbs`<SystemStatus::Status @isRunning={{true}} @isWorking={{false}} />`
+    );
+
+    assert
+      .dom('[system-status-status-loading]')
+      .exists()
+      .containsText('t:checking:()');
+  });
```

Comment:
test these scenarios using `<SystemStatus />` component and not the internal status component